### DISCUSSION TITLE
Need to see my private repos in the dashboard

### DISCUSSION THREAD

**Discussion by NickLennonLiu** [community/community#122226] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

When I go to github.com, I have to click on my profile picture and click my repositories to navigate to find my private repos, however in a lot of times they are the ones that I frequently check, so is there a way to see my private repos in my github dashboard?

**Comment by davevad93** [comment, marked as answer]
Hi @NickLennonLiu ,

I'm not aware of a way to view directly on the dashboard your private repos, however there's a quicker way to access them rather than going to the `Repositories` window; on the left side the dashboard there's the option to directly search for a repo,  just type the name and access it from there:

![00](https://github.com/community/community/assets/94308234/882f3bf0-4f3c-4597-b240-0d60b7fef20d)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @NickLennonLiu !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the `Feed` category is solely related to conversations around the GitHub product `Feed`. This question should be in the Repositories category. I've gone ahead and moved it for you. Good luck!
